configs/essentials.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  extends: [
+    '../rules/best-practices',
+    '../rules/errors',
+    '../rules/es6',
+    '../rules/imports',
+    '../rules/promise',
+    '../rules/style',
+    '../rules/variables',
+  ],
+  parserOptions: {
+    ecmaVersion: 12,
```

Comment:
Version 3 will support [ESLint 7.32.0](https://github.com/eslint/eslint/releases/tag/v7.32.0).
ESLint 7.32.0 was released in July 2021, so specify 12 (=2021) for `ecmaVersion`.

ref: [Configure Language Options - ESLint - Pluggable JavaScript Linter](https://eslint.org/docs/latest/use/configure/language-options#specifying-environments)